xapian-applications/omega/omindex.cc

```diff
@@ -133,6 +120,20 @@ index_file(const string &file, const string &url, DirectoryIterator & d,
 	return;
     }
 
+    // if can't get the mime type from extension,call libmagic to get it
```

Comment:
One small but quite important point - you should aim to make comments clear and unambiguous, and getting the grammar and punctuation helps that (though I realise that's often harder for non-native speakers).

So better to capitalise the sentence, give the verb a subject ("we can't" rather than "can't"), put a space after the comma, and a full stop (period) at the end.  I've pushed a commit to fix these in this case.